webdistill.css

```
@@ -1,4 +1,41 @@
-/*@media screen and (min-width: 768px)*/
+@media screen and (max-width: 900px) {
+	.deal2 {
+		height: 400px;
+		display: none;
+		border: 2pt solid black;

```

Use `px` anytime you're doing anything absolute. If it needs to scale with the type, use `rem`. 

```suggestion
		border: 2px solid black;

```

---

TLDR: Never use `pt` for anything.